docker/pulsar/build-scripts/remove-unnecessary-native-binaries.sh

```diff
@@ -41,9 +41,13 @@ unzip $ROCKSDB_JAR > /dev/null
 
 if [ "$ARCH" = "x86_64" ]
 then
-  ROCKSDB_TO_KEEP=linux64-musl
+  ROCKSDB_TO_KEEP=linux64
 else
-  ROCKSDB_TO_KEEP=linux-$ARCH-musl
+  ROCKSDB_TO_KEEP=linux-$ARCH
+fi
+
+if [ "$1" != "glibc" ]; then
+  ROCKSDB_TO_KEEP="${ROCKSDB_TO_KEEP}-musl"
```

Comment:
Do you miss arch check?

Reply:
There's no need for the arch check here. This appends `-musl` to `ROCKSDB_TO_KEEP` unless `glibc` is passed as the first argument to the script. This is to retain the previous behavior while supporting build of `glibc` based image.
In `Dockerfile.wolfi`, it will call this script with `glibc` parameter.